docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,128 @@
+</td>
+<td style="width:50%" align="center">
+<img src="./media/view_Figure_Scale.gif" width="100%">
+</td>
+</tr>
+</table>
 
-The following table lists the basic event provided by the `View` class:
+
+<a name="2"></a>
+## Directional Navigation
+
+NUI also provides directional navigation between `Views` by using arrow keys on the keyboard. Developer can simply set following properties to specify next `FocusableViews` for each direction:
+
+[View.UpFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_UpFocusableView)
+
+[View.DownFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_DownFocusableView)
+
+[View.LeftFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_LeftFocusableView)
+
+[View.RightFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_RightFocusableView)
+
+When a arrow key is pressed and the current `View` has next `FocusableView` for the direction, system passes focus to the next `FocusableView`. However, if the current `View` doesn't have next `FocusableView` for the direction or the next `ForcusableView` is not on the window, the property will return `NULL` and nothing will happen.
+
+
+<a name="3"></a>
+## View Event
+
+`View` provides veriety `events` commonly used by the components. The following table lists the basic event provided by the `View`:
 
 **Table: View events**
 
-| Input signal        | Description                                               |
+| Event               | Description                                               |
 | ------------------- | --------------------------------------------------------- |
-| `AddedToWindow`     | Emitted after the view has been connected to the window.  |
+| `KeyEvent`          | Emitted when the key event is received.                   |
+| `TouchEvent`        | Emitted when the touch input is received.                 |
+| `HoverEvent`        | Emitted when the hover input is received.                 |
+| `WheelEvent`        | Emitted when the wheel event is received.                 |
 | `FocusGained`       | Emitted when the control gets the key input focus.        |
 | `FocusLost`         | Emitted when the control loses key input focus.           |
-| `HoverEvent`        | Emitted when the hover input is received.                 |
-| `KeyEvent`          | Emitted when the key event is received.                   |
-| `LayoutDirectionChanged` | Emitted when  the layout direction property of this or a parent view is changed. |
 | `Relayout`          | Emitted after the size has been set on the view during relayout. |
+| `LayoutDirectionChanged` | Emitted when  the layout direction property of this or a parent view is changed. |
```

Comment:
I think "the" is better then child, because, a subject that handle these events is `View`.
So i think 'the View' and its parent View is better.

Thank for your good comment.